PyPoE/cli/exporter/wiki/parsers/lua.py

```diff
@@ -506,10 +506,8 @@ def main(self, parsed_args):
             self._copy_from_keys(
                 row, self._COPY_KEYS_BESTIARY_COMPONENTS, components
             )
-            if row['RarityKey'] != RARITY.ANY:
-                components[-1]['rarity'] = self.rr['ClientStrings.dat'].index[
-                    'Id']['ItemDisplayString' + row['RarityKey'].name_upper][
-                    'Text']
+            if row['BeastRarity'] != RARITY.ANY:
+                components[-1]['rarity'] = self.rr['ClientStrings.dat'].index['Id']['ItemDisplayString' + row['BeastRarity'].name_upper]['Text']
```

Comment:
This one's a doozy to follow by-eye. Can we try:

```py
if row['BeastRarity'] != RARITY.ANY:
    display_string = 'ItemDisplayString' + row['BeastRarity'].name_upper
    components[-1]['rarity'] = self.rr['ClientStrings.dat'].index['Id'][display_string]['Text']
```

Reply:
Sure 👍